authnz/src/main/java/io/stargate/auth/Credentials.java

```diff
@@ -0,0 +1,20 @@
+package io.stargate.auth;
+
+public class Credentials {
+
+  private final String username;
+  private final String password;
```

Comment:
nit: if we want to be strict, it might be worth converting this to `char[]` so that the password could be zeroed-out after use. This is the pattern in the JRE internal security APIs. Doing this is moderately important in Stargate, I guess, since we allow arbitrary OSGi bundles to be plugged in.

Reply:
cf. `javax.security.auth.callback.PasswordCallback`

Reply:
Done